testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthApplicationStartupHTTPEndpointTestCase.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.test.integration.microprofile.health;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.jboss.as.arquillian.container.ManagementClient;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import javax.json.JsonValue;
+import java.io.IOException;
+import java.io.StringReader;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * @author <a href="http://xstefank.io/">Martin Stefanko</a> (c) 2021 Red Hat inc.
+ */
+public class MicroProfileHealthApplicationStartupHTTPEndpointTestCase extends MicroProfileHealthApplicationStartupTestBase {
+
+    void checkGlobalOutcome(ManagementClient managementClient, String operation, boolean mustBeUP, String probeName) throws IOException {
+
+        assertEquals("check-started", operation);
+        final String httpEndpoint = "/health/started";
+
+        final String healthURL = "http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + httpEndpoint;
+
+        try (CloseableHttpClient client = HttpClients.createDefault()) {
+
+            CloseableHttpResponse resp = client.execute(new HttpGet(healthURL));
+            String content = EntityUtils.toString(resp.getEntity());
+            resp.close();
+
+            assertEquals(content, mustBeUP ? 200 : 503, resp.getStatusLine().getStatusCode());
```

Comment:
yes, I wanted to do that but I noticed that in other readiness and liveness tests too so I would keep it here and I'll fix it in all tests in follow if that's ok?

Reply:
Ok, thanks a lot @xstefank, that is a great idea indeed to have an issue tracker for this follow up task. Can you please copy/paste a link for the new issue in here once that it has been created, so that we can track where it all started from in the future?

Reply:
ok, here it is https://issues.redhat.com/browse/WFLY-15114